modules/oadp-self-service-admin-spec-enforce-nabsl.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// backup_and_restore/application_backup_and_restore/oadp-self-service/oadp-self-service-cluster-admin-use-cases.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-self-service-admin-spec-enforce-nabsl_{context}"]
+= Self-Service administrator spec enforcement for NABSL
+
+As a cluster administrator, you can enforce the following fields for a `NonAdminBackupStorageLocation` (NABSL) CR:
+
+* `objectStorage`
+* `credential`
+* `config`
+* `accessMode`
+* `validationFrequency`
+
+For example, if you want to enforce a namespace admin user to use a specific storage bucket, you can set up the `DataProtectionApplication` (DPA) CR as following:
+
+.Example `DataProtectionApplication` CR
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+...
+spec:
+  nonAdmin:
+    enable: true
+    enforceBSLSpec: # <1>
+      config: # <2>
+        checksumAlgorithm: ""
+        profile: default
+        region: us-west-2
+      objectStorage: # <3>
+        bucket: my-company-bucket
+        prefix: velero 
+      provider: aws  
+----
+
+In the example DPA, you have:
+
+* Added the `enforceBSLSpec` section as shown in annotation *1*.
+* Enforced the `config` section of a NABSL to use an {aws-short} S3 bucket in the us-west-2 region as shown in annotation *2*. 
+* Enforced the `objectStorage` section of a NABSL to use a company bucket named `my-company-bucket` as shown in annotation *3*.
```

Comment:
This way of using code callouts is incorrect. You can use code callouts or create a description/definition list. But both options have specific style rules that need to be followed. See https://redhat-documentation.github.io/modular-docs/#creating-reference-modules